src/main/resources/edu/cornell/kfs/module/purap/document/workflow/JaggaerRoleLinkMappingMaintenanceDocument.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+    <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+        <documentType>
+            <name>JRLM</name>
```

Comment:
sorry for not mentioning this before, but rather than ingesting XML, could this doc type be done by inserting a new record into kfs.krew_doc_typ_t?

Reply:
Yes, based on my review of Nancy's changes on a different PR, I believe I could insert the doc type with SQL just by inserting a record into KREW_DOC_TYP_T, due to the type of setup being inherited by the new doc type. I will look into adding that SQL.

Reply:
Can this be removed if you are inserting the document type with SQL?

Reply:
For our Cornell-specific doc types, I thought we still kept their workflow XML in our codebase for reference, even when using SQL to insert them manually.

Reply:
hmmm, I would think the reverse, lets check with the team

Reply:
based on discussion in teams, we do want to keep this